responsive-article/gulcin/index.html

```diff
@@ -0,0 +1,275 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Medium Article</title>
+  <link href="styles.css" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/8a55a6124d.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet"
+    href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat&family=Pathway+Extreme:wght@400;600&display=swap"
+    rel="stylesheet">
+</head>
+
+<body>
+  <header>
+    <nav>
+      <div class="left-nav">
+        <div class="medium-icon">
+          <a href="#"><img src="images/iconn.jpg" alt="Medium Icon"></a>
+        </div>
+        <div class="search">
+          <i class="fa-solid fa-magnifying-glass"></i>
+          <label for="nav-search"></label>
+          <input type="search" id="nav-search" name="nav-search" placeholder="Search Medium">
+        </div>
+      </div>
+      </div>
+      <ul class="right-nav">
+        <li class="nav-li orderlast"><i class="fa-regular fa-pen-to-square"></i><a href="#">Write</a></li>
+        <li class="nav-li selected"><a class="selected-a" href="#">Sign Up</a></li>
+        <li class="nav-li"><a href="#">Sign In</a></li>
+        <li class="nav-li account-symbol"><a href="#"><span class="material-symbols-outlined">
+              account_circle
+            </span></a></li>
+      </ul>
+      <div class="no-display">
+        <a href="#">Open in App</a><span class="material-symbols-outlined">
+          north_east
+        </span>
+      </div>
+    </nav>
+  </header>
+  <main>
+    <article class="main-article">
+      <h1>Time-saving CSS techniques to create responsive images</h1>
+      <div class="user-top">
+        <img class="user-img" src="images/iconn.jpg" alt="user picture">
+        <div class="user-text"><a href="#" class="underline">Andry Zaganel</a><a href="#" class="user-text-a">Follow</a></div>
+        <div class="user-text1">
+          <span>Published in &nbsp;<a href="#" class="underline" >We've moved to freeCodeCamp.org/news</a><span>&nbsp; &nbsp; &nbsp; 3 min read
+              &nbsp; &nbsp; &nbsp; &nbsp;Aug 12, 2018</span>
+        </div>
+      </div>
+      <div class="icons">
+        <div>
+          <a href="#"><i class="fa-solid fa-hands-clapping"></i></a><span>10.4K</span>
+          <a href="#"><i class="fa-regular fa-comment"></i></a><span>21</span>
+        </div>
+        <div>
+          <a href="#"><i class="fa-regular fa-bookmark"></i></a>
```

Comment:
You can keep it as it this time, but when you see elements like this, often an `ul` is used in HTML to put them together.